dali/kernels/normalize/normalize_cpu.h

```diff
@@ -156,20 +171,20 @@ struct NormalizeCPU {
 
       // last dimension - 1D case, which can be either:
       if (param_strides[axis] == 0)
-        normalize(out, in, data_shape_[axis], *mean, *stddev);  // shared factors or..
+        normalize(out, in, data_shape_[axis], *mean, *scale, shift_);  // shared factors or..
```

Comment:
Just a thought: if you rename `inv_sttdev` to `scale` maybe you could rename `mean` to `bias`?